inference-engine/src/readers/ir_reader/ie_ir_parser.hpp

```diff
@@ -251,6 +256,36 @@ class V10Parser : public IParser {
             stringToType<double>(val, value);
             adapter.set(value);
         }
+        void on_adapter(const std::string& name, ngraph::ValueAccessor<void*>& adapter) override  {
+            std::vector<int64_t> shape;
+            std::string el_type_str;
+            size_t offset;
+            size_t size;
+            pugi::xml_node dn = node.child("data");
```

Comment:
In general constant op uses this [adapter](https://github.com/openvinotoolkit/openvino/blob/master/ngraph/core/src/op/constant.cpp#L628) to read value.

I think that first of all we need to try to read the attribute with original name (value for constant) and only if we cannot find attribute with the original name in IR check that original name is "value" and if it is true try to read offset and size.

Reply:
Why validate value attribute as it does not appear in IR xml file?

Reply:
In general we had some thoughts about that we can use value attribute for small constants.

But in general this adapter can be used not only for constant op (for custom ops for example), and in order to avoid some unexpected cases we should try to read the attribute with original name.